index.js

```diff
@@ -122,6 +127,24 @@ function objEquiv(a, b, opts) {
     }
   }
 
+  // Maps
+  if (typeof Map && a.constructor === Map && b.constructor === Map) {
+    ka = Array.from(a.keys());
```

Comment:
`Array.from` is not necessarily available; we'll need to use a package or manually implement a while loop to call `.next()` on the iterator. However, not all Map implementations have `keys` either, so it's actually a pretty hard problem to robustly iterate Maps and Sets.